handsome-hedgehogs/workforce-management
# Em home

Ticket #15 

I made the nav bar links correct, added some information to the DB Initializer, and updated the Employee model to have StartDate.

In order to test:
1. Pull down the branch. 
2. Delete your database.
3. Run `Add-Migration AddStartDateToEmployee` in your powershell terminal.
4. Run `Update-Database` in your terminal.
5. Run the program, click the links in the nav bar, make sure that they all take you to the correct place and displays the appropriate data.
